greeting-service/src/test/java/dev/snowdrop/example/GreetingTest.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2016-2017 Red Hat, Inc, and individual contributors.
```

Comment:
2021 ? I'm not sure if you are too strict with headers though.

Reply:
Updated the dates.